pom.xml

```diff
@@ -6,7 +6,7 @@
 	<artifactId>lunatic-model</artifactId>
 	<packaging>jar</packaging>
 
-	<version>2.3.2-rc8</version>
+	<version>2.3.2-rc10</version>
```

Comment:
maybe switch to a proper new version? 2.4.+? I think that other branches of Lunatic-Model won't get higher than 2.3.Z before we make the 3.0.0 version (after Eno 3 release)